woocommerce/woocommerce-ios
# [Mobile Payments] Update URLs for card reader purchase and user guide

Closes: #5831 


### Description

Update the purchase URL and the link to the user guide, to point to those corresponding to the Stripe M2 reader.

 For context see pdfdoF-kx-p2
### Testing instructions

Log in to an eligible store, navigate to Settings > In-Person Payments.

1. Tap Order Card Reader. The app should present a web view, that will navigate to a 404 error, as the actual purchase url has not ben published yet.
2. Tap Card Reader Manual. The app should present a modal view with the Stripe M2 user guide

---
